### PR TITLE
Feature/winter 21  add sessions

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
~~Place holder data is incorporated (Refer to [Case Management Sample Data document]~~(https://quip.com/cbFcAPJF8t0z))
~~Base axe-core JEST test included for any new LWC (No critical violations)~~
~~CRUD/FLS is enforced in Apex~~
~~Update & Test permission sets to account for new features and CRUD/FLS~~
- [x] All modified classes are unit tested (Apex) at 100% coverage
- [x] UX approval or UX not necessary
- [x] PR contains draft release notes
- [x] Attach work item to the pull request with [Lurch](https://salesforce.quip.com/50ZRA5LEWVzH) `**Lurch:attach W-000000` or `**Lurch:add`
- [ ] All **acceptance criteria** have been met
    - [x] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by jjbennett** [comment]
[W-8013901](https://gus.lightning.force.com/a07B0000008IORLIA4)

File: force-app/main/default/classes/ServiceScheduleDomain.cls [hunk 1]
```diff
@@ -17,15 +17,21 @@ public with sharing class ServiceScheduleDomain {
         insert sObjects;
     }
 
-    private void updateSessionRelationships(ServiceScheduleModel model) {
+    private void setSessionDefaults(ServiceScheduleModel model) {
         for (ServiceSession__c session : model.serviceSessions) {
             session.ServiceSchedule__c = model.serviceSchedule.Id;
+            session.PrimaryServiceProvider__c = model.serviceSchedule.PrimaryServiceProvider__c;
+            session.OtherServiceProvider__c = model.serviceSchedule.OtherServiceProvider__c;
```
Comment (bethbrains) [inline force-app/main/default/classes/ServiceScheduleDomain.cls:24]: love doing this here. so much cleaner.

File: force-app/main/default/lwc/modal/modal.js [hunk 2]
```diff
@@ -41,8 +43,9 @@ export default class Modal extends LightningElement {
 
     get modalCss() {
         return (
-            "slds-modal slds-fade-in-open" +
-            (this.size === "fixed" ? "" : ` slds-modal_${this.size}`)
+            "slds-modal slds-fade-in-open slds-align_absolute-center" +
+            (this.size === "fixed" ? "" : ` slds-modal_${this.size}`) +
+            (this.isNested ? " nested-modal" : "")
```
Comment (bethbrains) [inline force-app/main/default/lwc/modal/modal.js:48]: clever!

File: force-app/main/default/lwc/reviewSessions/reviewSessions.js [hunk 3]
```diff
@@ -65,8 +81,16 @@ export default class ReviewSessions extends LightningElement {
...
+        return this._serviceScheduleModel.sessionFields.sessionStart.apiName;
+    }
+
+    get sessionEndFieldName() {
+        return this._serviceScheduleModel.sessionFields.sessionEnd.apiName;
+    }
+
     setLabels() {
-        this.objectName = this._serviceScheduleModel.labels.serviceSession.objectPluralLabel;
+        this.objectName = this._serviceScheduleModel.labels.serviceSession.objectApiName;
```
Comment (bethbrains) [inline force-app/main/default/lwc/reviewSessions/reviewSessions.js:93]: this was being used to display "x sessions" so we probably need another property for this.pluralobjectlabel or something?

**Review by bethbrains — CHANGES_REQUESTED** [review]
Looks great. 1 question.

**Review by bethbrains — APPROVED** [review]
Looks great.

**Review by AndersonTarren — APPROVED** [review]
Code Owner Approval

**Pull request merged**